main.go

```diff
@@ -257,6 +257,26 @@ func main() {
 	cmd.SetDir(workDir)
 
 	if err := cmd.Run(); err != nil {
+		fmt.Println()
+		log.Errorf("Fastlane command: (%s) failed", cmd.PrintableCommandArgs())
+		log.Errorf("See the error log below, and use it to send issue report to fastlane github issue tracker:")
```

Comment:
If you want to send an issue report to fastlane (https://github.com/fastlane/fastlane/issues/new), you can find the output of fastlane env (below):